src/styles.scss

```diff
@@ -3,6 +3,7 @@
 @import "~@ng-select/ng-select/themes/default.theme.css";
 @import '~@ngx-share/button/themes/default/default-theme';
 @import url('https://fonts.googleapis.com/css?family=Encode+Sans:400,500,600&display=swap');
+@import '~@angular/material/prebuilt-themes/indigo-pink.css';
```

Comment:
Does this affect prod env components in any way?

Reply:
No